### ISSUE TITLE
Could not find at Maven Central

### ISSUE THREAD

**Issue by agarwlGaurav** [pardom-zz/ActiveAndroid#113]
Wanted to add the library to my Android project, could not find the library when searched at Maven Central.

**Comment by vsigler** [comment]
Well, it's simply because it is not published there. :)

Just clone the repo and do 'mvn clean install'. Note that you will need either a running emulator or a live device connected for the tests to run.

Personally, I run my own repository on a VPS to host this as well as other library projects of mine.

**Comment by pardom-zz** [comment]
I have a Maven Central account setup for this, but haven't published it yet. I'll do it soon.

**Comment by johnhamelink** [comment]
I think it would be a good idea to push this to some sort of repository - I'm a gradle user so as far as I can tell I can't use your master branch as a source for my dependencies.

**Comment by markymc** [comment]
+1 for this.

**Comment by markmooibroek** [comment]
Hey @pardom, do you have some sort of an ETA on this? Would make my day =D Thanks for the great lib!

**Comment by orira** [comment]
Would make mine too once hosted, using gradle

**Comment by AaronRietschlin** [comment]
I agree, adding this to maven for use within Gradle would be very helpful. As I move my projects to gradle and Android Studio, I'd like to have as many of my dependencies using gradle as possible.

**Comment by nerdinand** [comment]
+1

**Comment by maveonair** [comment]
:+1:

**Comment by fer662** [comment]
pretty please?

**Comment by lkorth** [comment]
To add to this, it would be nice to not vendor android.jar and android-support-v4.jar. From what I can tell the only thing preventing this is the ant build file. Is there a reason all 3 build systems are present instead of just using one and releasing a jar for everyone else to use?

**Comment by matthewreimer** [comment]
+1

**Comment by jpotts18** [comment]
pretty pretty please :+1:

**Comment by rusfearuth** [comment]
+1

**Comment by olidroide** [comment]
+1 please add in Maven central

**Comment by johan-lindahl** [comment]
+1

**Comment by mazoni** [comment]
+1

**Comment by meierjan** [comment]
+1

**Comment by zjlovezj** [comment]
+1

**Comment by klingel** [comment]
+1

**Comment by se-rg-os** [comment]
+1 please add it to Maven Central. It is vital when using in projects which are based on powerful gradle.

**Comment by psamim** [comment]
+1

**Comment by Toolenaar** [comment]
+1

**Comment by marcferna** [comment]
+1

**Comment by malinkang** [comment]
+1

**Comment by rubensbrilhante** [comment]
+1

**Comment by andrecrispim** [comment]
+1

**Comment by ghost** [comment]
- 1

**Comment by Fi5t** [comment]
Do it dude!!!

**Comment by atov** [comment]
+1

**Comment by ganddev** [comment]
+1

**Comment by jmaffre** [comment]
please consider adding ActiveAndroid to jcenter as this is the new default repository used with Android's gradle plugin

**Comment by lkorth** [comment]
JCenter is a super set of Maven Central, if it's on Maven Central you'll be able to download it just fine.

**Comment by jmaffre** [comment]
Agree, but publishing on Maven Central can be cumbersome as oppose to Bintray...

**Comment by zty5678** [comment]
+1

**Comment by jitpack** [comment]
Btw, you can also get ActiveAndroid as a maven/gradle dependency from JitPack: https://jitpack.io/#pardom/ActiveAndroid/08c6335cd 

Using the latest commit id as the version here.

**Comment by vchatterji** [comment]
In the latest version of Android Studio (1.2.1.1)
Expand the Gradle Scripts node
Edit the build.gradle for your project (not module)
Add
allprojects {
    repositories {
        jcenter()
        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
    }
}
Edit the build.gradle for you module
Add compile 'com.michaelpardo:activeandroid:3.1.0-SNAPSHOT'
(in the dependencies node).
Should be good to go.
